vnext/Chakra/ChakraExecutor.cpp

```diff
@@ -510,21 +502,13 @@ void ChakraExecutor::loadApplicationScript(
   ReactMarker::logTaggedMarker(ReactMarker::RUN_JS_BUNDLE_START, scriptName.c_str());
 #endif
 
-  ChakraString jsSourceURL(sourceURL.c_str());
-
   // when debugging is enabled, don't use bytecode caching because ChakraCore
   // doesn't support it.
```

Comment:
I'm not convinced we still need this check.

Reply:
We're killing this Executor soon anyway though, so probably not worth touching this.

Reply:
We're killing this Executor soon anyway though, so probably not worth touching this.

---
In reply to: [341809035](https://github.com/microsoft/react-native-windows/pull/3584#discussion_r341809035) [](ancestors = 341809035)